### DISCUSSION TITLE
Not able to host website in github pages

### DISCUSSION THREAD

**Discussion by harish18092002** [community/community#70034] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I am facing an issue in pages when i launch my angular project repo in github my website does not hosted instead showing github documentation page...I tried this issue by ng build and setting dist folder as root but the issue continues 😢

**Comment by queenofcorgis** [comment]
Hi there - would you be able to share some more info like your repo, Page, or screenshots? That will help us better troubleshoot.

**Comment by daneedev** [comment, marked as answer]
Hello @harish18092002 ,
please make sure you setup the domain right in your repository settings, and check if your CNAME record is pointed to harish18092002.github.io

**Reply by harish18092002** [reply]
Ok sure ill check it thanks for your respond

**Reply by daneedev** [reply]
Ok, if that helped you mark this as accepted answer please.

**Comment by shyakachaste** [comment]
Hey @harish18092002 ,
check your domain if it was written well
